_pytest/deprecated.py

```diff
@@ -50,3 +50,9 @@ class RemovedInPytest4Warning(DeprecationWarning):
     "Metafunc.addcall is deprecated and scheduled to be removed in pytest 4.0.\n"
     "Please use Metafunc.parametrize instead."
 )
+
+PYTEST_PLUGINS_FROM_NON_TOP_LEVEL_CONFTEST = (
```

Comment:
in case its viable to use warnings.warn, lets please make this a warning object.
otherwise lets have a small todo comment

Reply:
done